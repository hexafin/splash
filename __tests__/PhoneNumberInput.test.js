import "react-native";
import React from "react";
import PhoneNumberInput from "../src/components/universal/PhoneNumberInput";

it("Phone Number Input renders correctly", () => {
	const snap = shallow(<PhoneNumberInput />);
	expect(snap).toMatchSnapshot();
});
