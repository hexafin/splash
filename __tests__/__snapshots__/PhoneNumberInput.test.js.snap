// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Phone Number Input renders correctly 1`] = `
<Component
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#FFFFFF",
      "borderRadius": 5,
      "flexDirection": "row",
      "height": 60,
      "shadowOffset": Object {
        "height": 5,
        "width": 0,
      },
      "shadowOpacity": 0.1,
      "shadowRadius": 10,
    }
  }
>
  <Component
    animationType="slide"
    hardwareAccelerated={false}
    transparent={false}
    visible={false}
  >
    <Component
      style={
        Object {
          "flex": 1,
          "flexDirection": "column",
          "justifyContent": "space-around",
          "padding": 40,
        }
      }
    >
      <Component
        style={
          Object {
            "color": "#333333",
            "fontSize": 22,
            "fontWeight": "600",
            "textAlign": "center",
          }
        }
      >
        Choose your country
      </Component>
      <Component
        style={
          Object {
            "backgroundColor": "#FFFFFF",
            "shadowOffset": Object {
              "height": 10,
              "width": 0,
            },
            "shadowOpacity": 0.1,
            "shadowRadius": 12,
          }
        }
      >
        <Picker
          mode="dialog"
          onValueChange={[Function]}
          selectedValue="United States"
        >
          <PickerItem
            key="countryPickerItem0"
            label="ðŸ‡¦ðŸ‡« Afghanistan"
            value="Afghanistan"
          />
          <PickerItem
            key="countryPickerItem1"
            label="ðŸ‡¦ðŸ‡± Albania"
            value="Albania"
          />
          <PickerItem
            key="countryPickerItem2"
            label="ðŸ‡©ðŸ‡¿ Algeria"
            value="Algeria"
          />
          <PickerItem
            key="countryPickerItem3"
            label="ðŸ‡¦ðŸ‡¸ American Samoa"
            value="American Samoa"
          />
          <PickerItem
            key="countryPickerItem4"
            label="ðŸ‡¦ðŸ‡© Andorra"
            value="Andorra"
          />
          <PickerItem
            key="countryPickerItem5"
            label="ðŸ‡¦ðŸ‡´ Angola"
            value="Angola"
          />
          <PickerItem
            key="countryPickerItem6"
            label="ðŸ‡¦ðŸ‡® Anguilla"
            value="Anguilla"
          />
          <PickerItem
            key="countryPickerItem7"
            label="ðŸ‡¦ðŸ‡¶ Antarctica"
            value="Antarctica"
          />
          <PickerItem
            key="countryPickerItem8"
            label="ðŸ‡¦ðŸ‡· Argentina"
            value="Argentina"
          />
          <PickerItem
            key="countryPickerItem9"
            label="ðŸ‡¦ðŸ‡² Armenia"
            value="Armenia"
          />
          <PickerItem
            key="countryPickerItem10"
            label="ðŸ‡¦ðŸ‡¼ Aruba"
            value="Aruba"
          />
          <PickerItem
            key="countryPickerItem11"
            label="ðŸ‡¦ðŸ‡¨ Ascension Island"
            value="Ascension Island"
          />
          <PickerItem
            key="countryPickerItem12"
            label="ðŸ‡¦ðŸ‡º Australia"
            value="Australia"
          />
          <PickerItem
            key="countryPickerItem13"
            label="ðŸ‡¦ðŸ‡¹ Austria"
            value="Austria"
          />
          <PickerItem
            key="countryPickerItem14"
            label="ðŸ‡¦ðŸ‡¿ Azerbaijan"
            value="Azerbaijan"
          />
          <PickerItem
            key="countryPickerItem15"
            label="ðŸ‡§ðŸ‡¸ Bahamas"
            value="Bahamas"
          />
          <PickerItem
            key="countryPickerItem16"
            label="ðŸ‡§ðŸ‡­ Bahrain"
            value="Bahrain"
          />
          <PickerItem
            key="countryPickerItem17"
            label="ðŸ‡§ðŸ‡© Bangladesh"
            value="Bangladesh"
          />
          <PickerItem
            key="countryPickerItem18"
            label="ðŸ‡§ðŸ‡§ Barbados"
            value="Barbados"
          />
          <PickerItem
            key="countryPickerItem19"
            label="ðŸ‡§ðŸ‡¾ Belarus"
            value="Belarus"
          />
          <PickerItem
            key="countryPickerItem20"
            label="ðŸ‡§ðŸ‡ª Belgium"
            value="Belgium"
          />
          <PickerItem
            key="countryPickerItem21"
            label="ðŸ‡§ðŸ‡¿ Belize"
            value="Belize"
          />
          <PickerItem
            key="countryPickerItem22"
            label="ðŸ‡§ðŸ‡¯ Benin"
            value="Benin"
          />
          <PickerItem
            key="countryPickerItem23"
            label="ðŸ‡§ðŸ‡² Bermuda"
            value="Bermuda"
          />
          <PickerItem
            key="countryPickerItem24"
            label="ðŸ‡§ðŸ‡¹ Bhutan"
            value="Bhutan"
          />
          <PickerItem
            key="countryPickerItem25"
            label="ðŸ‡§ðŸ‡´ Bolivia"
            value="Bolivia"
          />
          <PickerItem
            key="countryPickerItem26"
            label="ðŸ‡§ðŸ‡¼ Botswana"
            value="Botswana"
          />
          <PickerItem
            key="countryPickerItem27"
            label="ðŸ‡§ðŸ‡· Brazil"
            value="Brazil"
          />
          <PickerItem
            key="countryPickerItem28"
            label="ðŸ‡®ðŸ‡´ British Indian Ocean Territory"
            value="British Indian Ocean Territory"
          />
          <PickerItem
            key="countryPickerItem29"
            label="ðŸ‡§ðŸ‡« Burkina Faso"
            value="Burkina Faso"
          />
          <PickerItem
            key="countryPickerItem30"
            label="ðŸ‡§ðŸ‡® Burundi"
            value="Burundi"
          />
          <PickerItem
            key="countryPickerItem31"
            label="ðŸ‡°ðŸ‡­ Cambodia"
            value="Cambodia"
          />
          <PickerItem
            key="countryPickerItem32"
            label="ðŸ‡¨ðŸ‡² Cameroon"
            value="Cameroon"
          />
          <PickerItem
            key="countryPickerItem33"
            label="ðŸ‡¨ðŸ‡¦ Canada"
            value="Canada"
          />
          <PickerItem
            key="countryPickerItem34"
            label="ðŸ‡®ðŸ‡¨ Canary Islands"
            value="Canary Islands"
          />
          <PickerItem
            key="countryPickerItem35"
            label="ðŸ‡¨ðŸ‡» Cape Verde"
            value="Cape Verde"
          />
          <PickerItem
            key="countryPickerItem36"
            label="ðŸ‡°ðŸ‡¾ Cayman Islands"
            value="Cayman Islands"
          />
          <PickerItem
            key="countryPickerItem37"
            label="ðŸ‡¨ðŸ‡« Central African Republic"
            value="Central African Republic"
          />
          <PickerItem
            key="countryPickerItem38"
            label="ðŸ‡¹ðŸ‡© Chad"
            value="Chad"
          />
          <PickerItem
            key="countryPickerItem39"
            label="ðŸ‡¨ðŸ‡± Chile"
            value="Chile"
          />
          <PickerItem
            key="countryPickerItem40"
            label="ðŸ‡¨ðŸ‡³ China"
            value="China"
          />
          <PickerItem
            key="countryPickerItem41"
            label="ðŸ‡¨ðŸ‡½ Christmas Island"
            value="Christmas Island"
          />
          <PickerItem
            key="countryPickerItem42"
            label="ðŸ‡¨ðŸ‡´ Colombia"
            value="Colombia"
          />
          <PickerItem
            key="countryPickerItem43"
            label="ðŸ‡°ðŸ‡² Comoros"
            value="Comoros"
          />
          <PickerItem
            key="countryPickerItem44"
            label="ðŸ‡¨ðŸ‡° Cook Islands"
            value="Cook Islands"
          />
          <PickerItem
            key="countryPickerItem45"
            label="ðŸ‡¨ðŸ‡· Costa Rica"
            value="Costa Rica"
          />
          <PickerItem
            key="countryPickerItem46"
            label="ðŸ‡­ðŸ‡· Croatia"
            value="Croatia"
          />
          <PickerItem
            key="countryPickerItem47"
            label="ðŸ‡¨ðŸ‡º Cuba"
            value="Cuba"
          />
          <PickerItem
            key="countryPickerItem48"
            label="ðŸ‡¨ðŸ‡¿ Czech Republic"
            value="Czech Republic"
          />
          <PickerItem
            key="countryPickerItem49"
            label="ðŸ‡©ðŸ‡° Denmark"
            value="Denmark"
          />
          <PickerItem
            key="countryPickerItem50"
            label="ðŸ‡©ðŸ‡¯ Djibouti"
            value="Djibouti"
          />
          <PickerItem
            key="countryPickerItem51"
            label="ðŸ‡©ðŸ‡² Dominica"
            value="Dominica"
          />
          <PickerItem
            key="countryPickerItem52"
            label="ðŸ‡©ðŸ‡´ Dominican Republic"
            value="Dominican Republic"
          />
          <PickerItem
            key="countryPickerItem53"
            label="ðŸ‡ªðŸ‡¨ Ecuador"
            value="Ecuador"
          />
          <PickerItem
            key="countryPickerItem54"
            label="ðŸ‡ªðŸ‡¬ Egypt"
            value="Egypt"
          />
          <PickerItem
            key="countryPickerItem55"
            label="ðŸ‡¸ðŸ‡» El Salvador"
            value="El Salvador"
          />
          <PickerItem
            key="countryPickerItem56"
            label="ðŸ‡¬ðŸ‡¶ Equatorial Guinea"
            value="Equatorial Guinea"
          />
          <PickerItem
            key="countryPickerItem57"
            label="ðŸ‡ªðŸ‡· Eritrea"
            value="Eritrea"
          />
          <PickerItem
            key="countryPickerItem58"
            label="ðŸ‡ªðŸ‡ª Estonia"
            value="Estonia"
          />
          <PickerItem
            key="countryPickerItem59"
            label="ðŸ‡ªðŸ‡¹ Ethiopia"
            value="Ethiopia"
          />
          <PickerItem
            key="countryPickerItem60"
            label="ðŸ‡«ðŸ‡° Falkland Islands"
            value="Falkland Islands"
          />
          <PickerItem
            key="countryPickerItem61"
            label="ðŸ‡«ðŸ‡´ Faroe Islands"
            value="Faroe Islands"
          />
          <PickerItem
            key="countryPickerItem62"
            label="ðŸ‡«ðŸ‡¯ Fiji"
            value="Fiji"
          />
          <PickerItem
            key="countryPickerItem63"
            label="ðŸ‡«ðŸ‡® Finland"
            value="Finland"
          />
          <PickerItem
            key="countryPickerItem64"
            label="ðŸ‡«ðŸ‡· France"
            value="France"
          />
          <PickerItem
            key="countryPickerItem65"
            label="ðŸ‡¬ðŸ‡« French Guiana"
            value="French Guiana"
          />
          <PickerItem
            key="countryPickerItem66"
            label="ðŸ‡µðŸ‡« French Polynesia"
            value="French Polynesia"
          />
          <PickerItem
            key="countryPickerItem67"
            label="ðŸ‡¬ðŸ‡¦ Gabon"
            value="Gabon"
          />
          <PickerItem
            key="countryPickerItem68"
            label="ðŸ‡¬ðŸ‡² Gambia"
            value="Gambia"
          />
          <PickerItem
            key="countryPickerItem69"
            label="ðŸ‡¬ðŸ‡ª Georgia"
            value="Georgia"
          />
          <PickerItem
            key="countryPickerItem70"
            label="ðŸ‡©ðŸ‡ª Germany"
            value="Germany"
          />
          <PickerItem
            key="countryPickerItem71"
            label="ðŸ‡¬ðŸ‡­ Ghana"
            value="Ghana"
          />
          <PickerItem
            key="countryPickerItem72"
            label="ðŸ‡¬ðŸ‡® Gibraltar"
            value="Gibraltar"
          />
          <PickerItem
            key="countryPickerItem73"
            label="ðŸ‡¬ðŸ‡· Greece"
            value="Greece"
          />
          <PickerItem
            key="countryPickerItem74"
            label="ðŸ‡¬ðŸ‡± Greenland"
            value="Greenland"
          />
          <PickerItem
            key="countryPickerItem75"
            label="ðŸ‡¬ðŸ‡© Grenada"
            value="Grenada"
          />
          <PickerItem
            key="countryPickerItem76"
            label="ðŸ‡¬ðŸ‡µ Guadeloupe"
            value="Guadeloupe"
          />
          <PickerItem
            key="countryPickerItem77"
            label="ðŸ‡¬ðŸ‡º Guam"
            value="Guam"
          />
          <PickerItem
            key="countryPickerItem78"
            label="ðŸ‡¬ðŸ‡¹ Guatemala"
            value="Guatemala"
          />
          <PickerItem
            key="countryPickerItem79"
            label="ðŸ‡¬ðŸ‡¬ Guernsey"
            value="Guernsey"
          />
          <PickerItem
            key="countryPickerItem80"
            label="ðŸ‡¬ðŸ‡³ Guinea"
            value="Guinea"
          />
          <PickerItem
            key="countryPickerItem81"
            label="ðŸ‡¬ðŸ‡¼ Guinea-Bissau"
            value="Guinea-Bissau"
          />
          <PickerItem
            key="countryPickerItem82"
            label="ðŸ‡¬ðŸ‡¾ Guyana"
            value="Guyana"
          />
          <PickerItem
            key="countryPickerItem83"
            label="ðŸ‡­ðŸ‡¹ Haiti"
            value="Haiti"
          />
          <PickerItem
            key="countryPickerItem84"
            label="ðŸ‡­ðŸ‡³ Honduras"
            value="Honduras"
          />
          <PickerItem
            key="countryPickerItem85"
            label="ðŸ‡­ðŸ‡° Hong Kong"
            value="Hong Kong"
          />
          <PickerItem
            key="countryPickerItem86"
            label="ðŸ‡­ðŸ‡º Hungary"
            value="Hungary"
          />
          <PickerItem
            key="countryPickerItem87"
            label="ðŸ‡®ðŸ‡¸ Iceland"
            value="Iceland"
          />
          <PickerItem
            key="countryPickerItem88"
            label="ðŸ‡®ðŸ‡³ India"
            value="India"
          />
          <PickerItem
            key="countryPickerItem89"
            label="ðŸ‡®ðŸ‡© Indonesia"
            value="Indonesia"
          />
          <PickerItem
            key="countryPickerItem90"
            label="ðŸ‡®ðŸ‡· Iran"
            value="Iran"
          />
          <PickerItem
            key="countryPickerItem91"
            label="ðŸ‡®ðŸ‡¶ Iraq"
            value="Iraq"
          />
          <PickerItem
            key="countryPickerItem92"
            label="ðŸ‡®ðŸ‡ª Ireland"
            value="Ireland"
          />
          <PickerItem
            key="countryPickerItem93"
            label="ðŸ‡®ðŸ‡² Isle of Man"
            value="Isle of Man"
          />
          <PickerItem
            key="countryPickerItem94"
            label="ðŸ‡®ðŸ‡± Israel"
            value="Israel"
          />
          <PickerItem
            key="countryPickerItem95"
            label="ðŸ‡®ðŸ‡¹ Italy"
            value="Italy"
          />
          <PickerItem
            key="countryPickerItem96"
            label="ðŸ‡¯ðŸ‡² Jamaica"
            value="Jamaica"
          />
          <PickerItem
            key="countryPickerItem97"
            label="ðŸ‡¯ðŸ‡µ Japan"
            value="Japan"
          />
          <PickerItem
            key="countryPickerItem98"
            label="ðŸ‡¯ðŸ‡ª Jersey"
            value="Jersey"
          />
          <PickerItem
            key="countryPickerItem99"
            label="ðŸ‡¯ðŸ‡´ Jordan"
            value="Jordan"
          />
          <PickerItem
            key="countryPickerItem100"
            label="ðŸ‡°ðŸ‡¿ Kazakhstan"
            value="Kazakhstan"
          />
          <PickerItem
            key="countryPickerItem101"
            label="ðŸ‡°ðŸ‡ª Kenya"
            value="Kenya"
          />
          <PickerItem
            key="countryPickerItem102"
            label="ðŸ‡°ðŸ‡® Kiribati"
            value="Kiribati"
          />
          <PickerItem
            key="countryPickerItem103"
            label="ðŸ‡°ðŸ‡¼ Kuwait"
            value="Kuwait"
          />
          <PickerItem
            key="countryPickerItem104"
            label="ðŸ‡±ðŸ‡¦ Laos"
            value="Laos"
          />
          <PickerItem
            key="countryPickerItem105"
            label="ðŸ‡±ðŸ‡» Latvia"
            value="Latvia"
          />
          <PickerItem
            key="countryPickerItem106"
            label="ðŸ‡±ðŸ‡§ Lebanon"
            value="Lebanon"
          />
          <PickerItem
            key="countryPickerItem107"
            label="ðŸ‡±ðŸ‡¸ Lesotho"
            value="Lesotho"
          />
          <PickerItem
            key="countryPickerItem108"
            label="ðŸ‡±ðŸ‡· Liberia"
            value="Liberia"
          />
          <PickerItem
            key="countryPickerItem109"
            label="ðŸ‡±ðŸ‡¾ Libya"
            value="Libya"
          />
          <PickerItem
            key="countryPickerItem110"
            label="ðŸ‡±ðŸ‡® Liechtenstein"
            value="Liechtenstein"
          />
          <PickerItem
            key="countryPickerItem111"
            label="ðŸ‡±ðŸ‡¹ Lithuania"
            value="Lithuania"
          />
          <PickerItem
            key="countryPickerItem112"
            label="ðŸ‡±ðŸ‡º Luxembourg"
            value="Luxembourg"
          />
          <PickerItem
            key="countryPickerItem113"
            label="ðŸ‡²ðŸ‡° Macedonia"
            value="Macedonia"
          />
          <PickerItem
            key="countryPickerItem114"
            label="ðŸ‡²ðŸ‡¬ Madagascar"
            value="Madagascar"
          />
          <PickerItem
            key="countryPickerItem115"
            label="ðŸ‡²ðŸ‡¼ Malawi"
            value="Malawi"
          />
          <PickerItem
            key="countryPickerItem116"
            label="ðŸ‡²ðŸ‡¾ Malaysia"
            value="Malaysia"
          />
          <PickerItem
            key="countryPickerItem117"
            label="ðŸ‡²ðŸ‡» Maldives"
            value="Maldives"
          />
          <PickerItem
            key="countryPickerItem118"
            label="ðŸ‡²ðŸ‡± Mali"
            value="Mali"
          />
          <PickerItem
            key="countryPickerItem119"
            label="ðŸ‡²ðŸ‡¹ Malta"
            value="Malta"
          />
          <PickerItem
            key="countryPickerItem120"
            label="ðŸ‡²ðŸ‡­ Marshall Islands"
            value="Marshall Islands"
          />
          <PickerItem
            key="countryPickerItem121"
            label="ðŸ‡²ðŸ‡¶ Martinique"
            value="Martinique"
          />
          <PickerItem
            key="countryPickerItem122"
            label="ðŸ‡²ðŸ‡· Mauritania"
            value="Mauritania"
          />
          <PickerItem
            key="countryPickerItem123"
            label="ðŸ‡²ðŸ‡º Mauritius"
            value="Mauritius"
          />
          <PickerItem
            key="countryPickerItem124"
            label="ðŸ‡¾ðŸ‡¹ Mayotte"
            value="Mayotte"
          />
          <PickerItem
            key="countryPickerItem125"
            label="ðŸ‡²ðŸ‡½ Mexico"
            value="Mexico"
          />
          <PickerItem
            key="countryPickerItem126"
            label="ðŸ‡«ðŸ‡² Micronesia"
            value="Micronesia"
          />
          <PickerItem
            key="countryPickerItem127"
            label="ðŸ‡²ðŸ‡© Moldova"
            value="Moldova"
          />
          <PickerItem
            key="countryPickerItem128"
            label="ðŸ‡²ðŸ‡¨ Monaco"
            value="Monaco"
          />
          <PickerItem
            key="countryPickerItem129"
            label="ðŸ‡²ðŸ‡³ Mongolia"
            value="Mongolia"
          />
          <PickerItem
            key="countryPickerItem130"
            label="ðŸ‡²ðŸ‡ª Montenegro"
            value="Montenegro"
          />
          <PickerItem
            key="countryPickerItem131"
            label="ðŸ‡²ðŸ‡¸ Montserrat"
            value="Montserrat"
          />
          <PickerItem
            key="countryPickerItem132"
            label="ðŸ‡²ðŸ‡¦ Morocco"
            value="Morocco"
          />
          <PickerItem
            key="countryPickerItem133"
            label="ðŸ‡²ðŸ‡¿ Mozambique"
            value="Mozambique"
          />
          <PickerItem
            key="countryPickerItem134"
            label="ðŸ‡²ðŸ‡² Myanmar"
            value="Myanmar"
          />
          <PickerItem
            key="countryPickerItem135"
            label="ðŸ‡³ðŸ‡¦ Namibia"
            value="Namibia"
          />
          <PickerItem
            key="countryPickerItem136"
            label="ðŸ‡³ðŸ‡· Nauru"
            value="Nauru"
          />
          <PickerItem
            key="countryPickerItem137"
            label="ðŸ‡³ðŸ‡µ Nepal"
            value="Nepal"
          />
          <PickerItem
            key="countryPickerItem138"
            label="ðŸ‡³ðŸ‡± Netherlands"
            value="Netherlands"
          />
          <PickerItem
            key="countryPickerItem139"
            label="ðŸ‡³ðŸ‡¨ New Caledonia"
            value="New Caledonia"
          />
          <PickerItem
            key="countryPickerItem140"
            label="ðŸ‡³ðŸ‡¿ New Zealand"
            value="New Zealand"
          />
          <PickerItem
            key="countryPickerItem141"
            label="ðŸ‡³ðŸ‡® Nicaragua"
            value="Nicaragua"
          />
          <PickerItem
            key="countryPickerItem142"
            label="ðŸ‡³ðŸ‡ª Niger"
            value="Niger"
          />
          <PickerItem
            key="countryPickerItem143"
            label="ðŸ‡³ðŸ‡¬ Nigeria"
            value="Nigeria"
          />
          <PickerItem
            key="countryPickerItem144"
            label="ðŸ‡³ðŸ‡º Niue"
            value="Niue"
          />
          <PickerItem
            key="countryPickerItem145"
            label="ðŸ‡³ðŸ‡« Norfolk Island"
            value="Norfolk Island"
          />
          <PickerItem
            key="countryPickerItem146"
            label="ðŸ‡²ðŸ‡µ Northern Mariana Islands"
            value="Northern Mariana Islands"
          />
          <PickerItem
            key="countryPickerItem147"
            label="ðŸ‡³ðŸ‡´ Norway"
            value="Norway"
          />
          <PickerItem
            key="countryPickerItem148"
            label="ðŸ‡´ðŸ‡² Oman"
            value="Oman"
          />
          <PickerItem
            key="countryPickerItem149"
            label="ðŸ‡µðŸ‡° Pakistan"
            value="Pakistan"
          />
          <PickerItem
            key="countryPickerItem150"
            label="ðŸ‡µðŸ‡¼ Palau"
            value="Palau"
          />
          <PickerItem
            key="countryPickerItem151"
            label="ðŸ‡µðŸ‡¦ Panama"
            value="Panama"
          />
          <PickerItem
            key="countryPickerItem152"
            label="ðŸ‡µðŸ‡¬ Papua New Guinea"
            value="Papua New Guinea"
          />
          <PickerItem
            key="countryPickerItem153"
            label="ðŸ‡µðŸ‡¾ Paraguay"
            value="Paraguay"
          />
          <PickerItem
            key="countryPickerItem154"
            label="ðŸ‡µðŸ‡ª Peru"
            value="Peru"
          />
          <PickerItem
            key="countryPickerItem155"
            label="ðŸ‡µðŸ‡­ Philippines"
            value="Philippines"
          />
          <PickerItem
            key="countryPickerItem156"
            label="ðŸ‡µðŸ‡± Poland"
            value="Poland"
          />
          <PickerItem
            key="countryPickerItem157"
            label="ðŸ‡µðŸ‡¹ Portugal"
            value="Portugal"
          />
          <PickerItem
            key="countryPickerItem158"
            label="ðŸ‡µðŸ‡· Puerto Rico"
            value="Puerto Rico"
          />
          <PickerItem
            key="countryPickerItem159"
            label="ðŸ‡¶ðŸ‡¦ Qatar"
            value="Qatar"
          />
          <PickerItem
            key="countryPickerItem160"
            label="ðŸ‡·ðŸ‡´ Romania"
            value="Romania"
          />
          <PickerItem
            key="countryPickerItem161"
            label="ðŸ‡·ðŸ‡º Russia"
            value="Russia"
          />
          <PickerItem
            key="countryPickerItem162"
            label="ðŸ‡·ðŸ‡¼ Rwanda"
            value="Rwanda"
          />
          <PickerItem
            key="countryPickerItem163"
            label="ðŸ‡¼ðŸ‡¸ Samoa"
            value="Samoa"
          />
          <PickerItem
            key="countryPickerItem164"
            label="ðŸ‡¸ðŸ‡² San Marino"
            value="San Marino"
          />
          <PickerItem
            key="countryPickerItem165"
            label="ðŸ‡¸ðŸ‡¦ Saudi Arabia"
            value="Saudi Arabia"
          />
          <PickerItem
            key="countryPickerItem166"
            label="ðŸ‡¸ðŸ‡³ Senegal"
            value="Senegal"
          />
          <PickerItem
            key="countryPickerItem167"
            label="ðŸ‡·ðŸ‡¸ Serbia"
            value="Serbia"
          />
          <PickerItem
            key="countryPickerItem168"
            label="ðŸ‡¸ðŸ‡¨ Seychelles"
            value="Seychelles"
          />
          <PickerItem
            key="countryPickerItem169"
            label="ðŸ‡¸ðŸ‡± Sierra Leone"
            value="Sierra Leone"
          />
          <PickerItem
            key="countryPickerItem170"
            label="ðŸ‡¸ðŸ‡¬ Singapore"
            value="Singapore"
          />
          <PickerItem
            key="countryPickerItem171"
            label="ðŸ‡¸ðŸ‡° Slovakia"
            value="Slovakia"
          />
          <PickerItem
            key="countryPickerItem172"
            label="ðŸ‡¸ðŸ‡® Slovenia"
            value="Slovenia"
          />
          <PickerItem
            key="countryPickerItem173"
            label="ðŸ‡¸ðŸ‡§ Solomon Islands"
            value="Solomon Islands"
          />
          <PickerItem
            key="countryPickerItem174"
            label="ðŸ‡¸ðŸ‡´ Somalia"
            value="Somalia"
          />
          <PickerItem
            key="countryPickerItem175"
            label="ðŸ‡¿ðŸ‡¦ South Africa"
            value="South Africa"
          />
          <PickerItem
            key="countryPickerItem176"
            label="ðŸ‡°ðŸ‡· South Korea"
            value="South Korea"
          />
          <PickerItem
            key="countryPickerItem177"
            label="ðŸ‡ªðŸ‡¸ Spain"
            value="Spain"
          />
          <PickerItem
            key="countryPickerItem178"
            label="ðŸ‡±ðŸ‡° Sri Lanka"
            value="Sri Lanka"
          />
          <PickerItem
            key="countryPickerItem179"
            label="ðŸ‡¸ðŸ‡© Sudan"
            value="Sudan"
          />
          <PickerItem
            key="countryPickerItem180"
            label="ðŸ‡¸ðŸ‡· Suriname"
            value="Suriname"
          />
          <PickerItem
            key="countryPickerItem181"
            label="ðŸ‡¸ðŸ‡¿ Swaziland"
            value="Swaziland"
          />
          <PickerItem
            key="countryPickerItem182"
            label="ðŸ‡¸ðŸ‡ª Sweden"
            value="Sweden"
          />
          <PickerItem
            key="countryPickerItem183"
            label="ðŸ‡¨ðŸ‡­ Switzerland"
            value="Switzerland"
          />
          <PickerItem
            key="countryPickerItem184"
            label="ðŸ‡¸ðŸ‡¾ Syria"
            value="Syria"
          />
          <PickerItem
            key="countryPickerItem185"
            label="ðŸ‡¹ðŸ‡¼ Taiwan"
            value="Taiwan"
          />
          <PickerItem
            key="countryPickerItem186"
            label="ðŸ‡¹ðŸ‡¯ Tajikistan"
            value="Tajikistan"
          />
          <PickerItem
            key="countryPickerItem187"
            label="ðŸ‡¹ðŸ‡¿ Tanzania"
            value="Tanzania"
          />
          <PickerItem
            key="countryPickerItem188"
            label="ðŸ‡¹ðŸ‡­ Thailand"
            value="Thailand"
          />
          <PickerItem
            key="countryPickerItem189"
            label="ðŸ‡¹ðŸ‡± Timor-Leste"
            value="Timor-Leste"
          />
          <PickerItem
            key="countryPickerItem190"
            label="ðŸ‡¹ðŸ‡¬ Togo"
            value="Togo"
          />
          <PickerItem
            key="countryPickerItem191"
            label="ðŸ‡¹ðŸ‡° Tokelau"
            value="Tokelau"
          />
          <PickerItem
            key="countryPickerItem192"
            label="ðŸ‡¹ðŸ‡´ Tonga"
            value="Tonga"
          />
          <PickerItem
            key="countryPickerItem193"
            label="ðŸ‡¹ðŸ‡³ Tunisia"
            value="Tunisia"
          />
          <PickerItem
            key="countryPickerItem194"
            label="ðŸ‡¹ðŸ‡· Turkey"
            value="Turkey"
          />
          <PickerItem
            key="countryPickerItem195"
            label="ðŸ‡¹ðŸ‡² Turkmenistan"
            value="Turkmenistan"
          />
          <PickerItem
            key="countryPickerItem196"
            label="ðŸ‡¹ðŸ‡» Tuvalu"
            value="Tuvalu"
          />
          <PickerItem
            key="countryPickerItem197"
            label="ðŸ‡ºðŸ‡¬ Uganda"
            value="Uganda"
          />
          <PickerItem
            key="countryPickerItem198"
            label="ðŸ‡ºðŸ‡¦ Ukraine"
            value="Ukraine"
          />
          <PickerItem
            key="countryPickerItem199"
            label="ðŸ‡¦ðŸ‡ª United Arab Emirates"
            value="United Arab Emirates"
          />
          <PickerItem
            key="countryPickerItem200"
            label="ðŸ‡¬ðŸ‡§ United Kingdom"
            value="United Kingdom"
          />
          <PickerItem
            key="countryPickerItem201"
            label="ðŸ‡ºðŸ‡¸ United States"
            value="United States"
          />
          <PickerItem
            key="countryPickerItem202"
            label="ðŸ‡ºðŸ‡¾ Uruguay"
            value="Uruguay"
          />
          <PickerItem
            key="countryPickerItem203"
            label="ðŸ‡ºðŸ‡¿ Uzbekistan"
            value="Uzbekistan"
          />
          <PickerItem
            key="countryPickerItem204"
            label="ðŸ‡»ðŸ‡º Vanuatu"
            value="Vanuatu"
          />
          <PickerItem
            key="countryPickerItem205"
            label="ðŸ‡»ðŸ‡ª Venezuela"
            value="Venezuela"
          />
          <PickerItem
            key="countryPickerItem206"
            label="ðŸ‡ªðŸ‡­ Western Sahara"
            value="Western Sahara"
          />
          <PickerItem
            key="countryPickerItem207"
            label="ðŸ‡¾ðŸ‡ª Yemen"
            value="Yemen"
          />
          <PickerItem
            key="countryPickerItem208"
            label="ðŸ‡¿ðŸ‡² Zambia"
            value="Zambia"
          />
          <PickerItem
            key="countryPickerItem209"
            label="ðŸ‡¿ðŸ‡¼ Zimbabwe"
            value="Zimbabwe"
          />
        </Picker>
      </Component>
      <Button
        onPress={[Function]}
        primary={true}
        title="Done"
      />
    </Component>
  </Component>
  <TouchableWithoutFeedback
    onPress={[Function]}
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "#FFFFFF",
        "flex": 1,
        "flexDirection": "row",
        "height": 60,
        "justifyContent": "center",
        "paddingRight": 10,
      }
    }
  >
    <Component
      style={
        Object {
          "padding": 0,
        }
      }
    >
      <Component
        style={
          Object {
            "color": "#333333",
            "fontSize": 22,
            "padding": 10,
          }
        }
      >
        ðŸ‡ºðŸ‡¸
         +
        1
      </Component>
      <Component
        style={
          Array [
            undefined,
            Object {
              "display": "none",
            },
          ]
        }
      >
        <LoadingCircle
          color="purple"
          size={20}
        />
      </Component>
    </Component>
  </TouchableWithoutFeedback>
  <Component
    style={
      Object {
        "backgroundColor": "rgba(0,0,0,0.1)",
        "height": 60,
        "width": 1,
      }
    }
  />
  <TextInput
    allowFontScaling={true}
    autoFocus={false}
    keyboardType="number-pad"
    maxLength={12}
    onChangeText={[Function]}
    placeholder="### ### ####"
    style={
      Object {
        "alignItems": "center",
        "color": "#333333",
        "flex": 3,
        "flexDirection": "row",
        "fontSize": 22,
        "height": 60,
        "justifyContent": "center",
        "padding": 10,
      }
    }
    underlineColorAndroid="transparent"
    value=""
  />
</Component>
`;
